modules/internal/ChapelRange.chpl

```diff
@@ -1882,7 +1882,9 @@ proc _cast(type t, r: range(?)) where isRangeType(t) {
     if boundsChecking && isIntType(count.type) && count < 0 then
       HaltWrappers.boundsCheckHalt("With a negative count, the range must have a last index.");
 
-    const (start, end) = if count == 0 then (low, low - 1)
+    // This cast avoids C compile-time warnings if    (vvv)
```

Comment:
What's the `(vvv)`?

Reply:
Sorry that wasn't clear... they were meant to be down-arrows pointing to the specific cast being commented on (since the statement has another), but then I had to wrap the line to make them point to the right place (and failed to move them down a line which might've been clearer).  Or, since the comment did end up taking two lines maybe I should've used.

``` 
| | |
v v v
```

Reply:
Personally, I'd probably drop the arrows, but your call